examples/deployment_plan.go

```diff
@@ -18,7 +19,7 @@ func main() {
 		username             = os.Getenv("ONEVIEW_OV_USER")
 		password             = os.Getenv("ONEVIEW_OV_PASSWORD")
 		domain               = os.Getenv("ONEVIEW_OV_DOMAIN")
-		api_version          = 1000
+		api_version          = 1020
```

Comment:
API version 1200, not 1020. Change the PR title as well.

Reply:
Its 1020 for Image streamer.